src/ui/button/Button.tsx

```diff
@@ -4,37 +4,49 @@ import { Button as MuiButton, IconButton as MuiIconButton } from '@material-ui/c
 
 import csx from './Button.scss';
 
+const Btn = (props, children) => {
```

Comment:
???

Reply:
will be removed along with the removal of the MUI dependent component